lib/qesapdeployment.pm

```diff
@@ -595,7 +592,6 @@ sub qesap_execute_conditional_retry {
     foreach (qw(cmd logname error_string)) { croak "Missing mandatory $_ argument" unless $args{$_}; }
 
     my $verbose = $args{verbose} ? "--verbose" : "";
-    $args{cmd_options} //= '';
```

Comment:
it was a never used argument within the method implementation of in any method call. So I removed it